### PR TITLE
Multiple metrics

### PR DESCRIPTION
**Pull request by edancanning**
# Changes
* Please put your changes here

# Security Implications
* Please put any possible security-related concerns here. Feel free to @andykram if you're unsure. If none, please put "None"

### CODE REVIEW

File: .eslintrc.js
Comment (erikvorhes) [inline .eslintrc.js:73]: Are these relevant to this repo?

**Review by erikvorhes — COMMENTED** [review]
EXCITING

File: .eslintrc.js
Comment (edancanning) [reply .eslintrc.js:73]: No idea how that got in...

Comment (erikvorhes) [reply .eslintrc.js:73]: `copypasta`

**Review by erikvorhes — APPROVED** [review]
Approving, assuming those questionable lint rules are excised before merging. :)

File: .eslintrc.js
Comment (edancanning) [reply .eslintrc.js:73]: Oh... it wasn't me... I won't name names though...

Comment (erikvorhes) [reply .eslintrc.js:73]: Wild, they don't seem to be in the file any more. WTH

**Pull request merged**
